docs/source/faq.rst

```diff
@@ -9,6 +9,18 @@ answers.
         :local:
         :backlinks: none
 
+General
+=======
+
+Does SimpleITK offer all the functionality as the C++ ITK?
+----------------------------------------------------------
+
+SimpleITK supports most ITK filters (:ref:`see list <lbl_filter>`) and the
```

Comment:
"most ITK filters" -> "most ITK image filters" and plug our image IO too.

Reply:
Done.